### DISCUSSION TITLE
v0.1.390

### DISCUSSION THREAD

**Discussion by DamianMorozov** [DamianMorozov/OpenTgResearcher#12] [Announcements]
### Added
- Set file date and time
- Scanning subdirectories for downloaded files to move them to the root directory
### Changed
- Combining source ID and user name settings

<hr /><em>This discussion was created from the release <a href='https://github.com/DamianMorozov/TgDownloader/releases/tag/v0.1.390'>v0.1.390</a>.</em>

**Comment by godlonely** [comment]
When I use "Connect the client to TG server" ,then input "API hash" and "APi id" , click 'Enter' key ,the program GG... I don't know what wrong with it . I use the VPN with the internet.

**Reply by DamianMorozov** [reply]
Can you attach a screenshot of your problem?

**Reply by godlonely** [reply]
And my 'hash' and 'id' is OK, I can use it to log in with other tools. Is it because I'm logged into the Asian region?

**Reply by DamianMorozov** [reply]
Sorry, but I can not see your screenshot.
![image](https://user-images.githubusercontent.com/13628284/212866679-93f11a50-cf77-4eb6-b2d2-545422d52260.png)

**Reply by godlonely** [reply]
![Snipaste_2023-01-17_17-11-37](https://user-images.githubusercontent.com/14304355/213070618-addb1cae-ef7f-4c8c-8ac1-c17051a0c961.png)
i don't know if it can help you

**Reply by DamianMorozov** [reply]
You may need to use a proxy to connect. I'm working on that right now.

**Reply by DamianMorozov** [reply]
> And my 'hash' and 'id' is OK, I can use it to log in with other tools. Is it because I'm logged into the Asian region?

Try to use the release [v0.1.500](https://github.com/DamianMorozov/TgDownloader/releases/tag/v0.1.500).
